release/create_candidate_pr.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=redefined-outer-name, invalid-name
+
+"""
+Creates a new release candidate branch and makes a pull request
+for it into the 'release' branch.
+"""
+
+from __future__ import print_function
+
+import argparse
+
+from release.get_token import get_token
+from release.github_api import GithubApi, RequestFailed
+from release import utils
+
+
+def _build_parser():
+    """ Creates an option parser for this command """
```

Comment:
That makes sense to me, is that something we can enforce automatically? I like it when robots tell me about things like that since otherwise they tend to be easy to forget (or y'know, not know about in the first place).